app/assets/stylesheets/components/_button-holder.css

```diff
@@ -0,0 +1,8 @@
+.button-holder {
```

Comment:
**question:** is this a button variation? If so, should we name it `button--holder`?

Reply:
no, it really is a button holder, a div that holds the buttons 😬